modules/assisted-installer-assisted-installer-prerequisites.adoc

```diff
@@ -0,0 +1,60 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='assisted-installer-prerequisites_{context}']
+= Assisted Installer prerequisites
+
+Successful installation with the {ai-full} requires ensuring the following prerequisites. The installer will validate these prerequisites before installation.
```

Comment:
The [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#installer-program) lists "the installer" as an incorrect form. 

Please refer to my previous comment regarding the ISG and future tense.